app/Response/Graph.hs

```diff
@@ -16,42 +16,8 @@ graphResponse =
                 (do
                     header "graph"
                     H.div ! A.id "container" $ do
+                        -- TODO: remove this inner div
                         H.div ! A.id "react-graph" ! A.class_ "react-graph" $ ""
-                        sideBar
                     disclaimer
                 )
                 graphScripts
-
-sideBar :: H.Html
-sideBar = do
-    H.div ! A.id "sidebar" $ do
-        H.div ! A.id "fce" $ do
-            H.div ! A.id "fcecount" $ "FCE Count: 0.0"
-            H.button ! A.id "reset" $ "Reset Graph"
-        H.nav ! A.id "sidebar-nav" $ H.ul $ do
-            H.li ! A.id "graphs-nav" $
-                H.a ! A.href "" $ "Graphs"
```

Comment:
Why is this an <a> tag?

Reply:
I'm not sure if you're asking me, but I think it's because there's supposed to be an onclick callback here.

Reply:
Right now this is causing the page to refresh when clicked on. Can I change this to a `div`?